setup-buildengine.sh

```diff
@@ -524,9 +520,11 @@ Creating $OPT_VM, compression=$OPT_COMPRESSION
  Release:     $OPT_RELEASE
  MerSDK VDI:  $OPT_VDI
  Tooling:     $OPT_TOOLING
- ARM target:  $OPT_TARGET_ARM
- i486 target: $OPT_TARGET_I486
 EOF
+for targetarch in ${!OPT_TARGET_FILENAMES[*]} ; do
+    echo "${targetarch^} target: ${OPT_TARGET_FILENAMES[$arch]}"
```

Comment:
Shouldn't this be `echo "${targetarch^} target: ${OPT_TARGET_FILENAMES[$targetarch]}"` ?